helm-charts/values.yaml

```diff
@@ -5,15 +5,47 @@
 replicaCount: 1
 
 image:
-  repository: nginx
-  pullPolicy: IfNotPresent
+  repository: caleydo
+  pullPolicy: Always
   # Overrides the image tag whose default is the chart appVersion.
   tag: ""
 
 imagePullSecrets: []
 nameOverride: ""
 fullnameOverride: ""
 
+#Variables
+MongoDBName: &MongoDBName "ordinoDB"
+MongoDBUser: &MongoDBUser "ordinoAdmin"
+MongoDBPass: &MongoDBPass "ordinoAdmin"
```

Comment:
Other question: could we provide this as k8s secret?

Reply:
It can be a secret but there is no need for it since it is only used in the configuration json file which is still kept as plaintext. If there is a way for Ordino to use such values as environment variables then I can move them to secrets and add them in the container environment through the deployment.

Reply:
Can you please still make it secrets?
This is less about the technical reasons but if our security find a password that is not using the methods provided then we have to go thru a pile of forms and part of that will be that we will have to forward them to you so you have to argue why it's not necessary.
Please, please, please! If it smells like anything close to credentials make it a secret.